packages/react-codemirror/src/lang-cypher/langCypher.ts

```diff
@@ -19,20 +20,30 @@ const facet = defineLanguageFacet({
 
 export type CypherConfig = {
   lint?: boolean;
+  featureFlags?: {
+    signatureInfoOnAutoCompletions?: boolean;
+    consoleCommands?: boolean;
+  };
   schema?: DbSchema;
   useLightVersion: boolean;
   setUseLightVersion?: (useLightVersion: boolean) => void;
 };
 
 export function cypher(config: CypherConfig) {
-  _internalFeatureFlags.consoleCommands = true;
+  const featureFlags = config.featureFlags;
+  // We allow to override the consoleCommands feature flag
+  if (featureFlags.consoleCommands !== undefined) {
```

Comment:
should all feature flags be handled here? `signatureInfoOnAutoCompletions` and any future ones we add

Reply:
It depends on whether we want to pass them down the language support or not. Meaning there are things we might want to block at codemirror level, but not necessarily at language support one. For example the signature info it's not needed to gate it at language support level for example.